cpp/pagerank_module/pagerank_module.cpp

```diff
@@ -35,9 +36,8 @@ void PagerankWrapper(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *resu
 
     auto graph = mg_utility::GetGraphView(memgraph_graph, result, memory, mg_graph::GraphType::kDirectedGraph);
 
-    const auto &graph_edges = graph->Edges();
+    auto graph_edges = graph->Edges();
     std::vector<pagerank_alg::EdgePair> pagerank_edges;
-    pagerank_edges.reserve(graph_edges.size());
```

Comment:
What is the rational behind this?

I think the old implementation is better, because:
1. it doesn't copy the vector returned by `graph->Edges()`
2. it possibly preallocates enough memory to reduce allocations

Am I missing something?